test/e2e/search_test.go

```diff
@@ -237,7 +237,7 @@ registries = ['{{.Host}}:{{.Port}}']`
 	})
 
 	It("podman search attempts HTTP if registry is in registries.insecure and force secure is false", func() {
-		SkipIfRemote("FIXME This should work on podman-remote")
+		SkipIfRemote("--tls-verify is not supportedon podman-remote search")
```

Comment:
[not worth re-pushing; please fix only if something else merits it] : "supportedon" needs space. Also lines 281, 320 below.